### ISSUE TITLE
Failing Test on zef install

### ISSUE THREAD

**Issue by masukomi** [croservices/cro-tls#13]
```
===> Testing: Cro::TLS:ver<0.8.8>:auth<zef:cro>
[Cro::TLS] # Failed test 'Listening for connections once the Supply is tapped'
[Cro::TLS] # at t/tls.t line 39
[Cro::TLS] # Server certificate verification failed: certificate has expired
[Cro::TLS] No such method 'close' for invocant of type 'Any'.  Did you mean
[Cro::TLS] 'clone'?
[Cro::TLS]   in block <unit> at t/tls.t line 44
===> Testing [FAIL]: Cro::TLS:ver<0.8.8>:auth<zef:cro>
Aborting due to test failure: Cro::TLS:ver<0.8.8>:auth<zef:cro> (use --force-test to override)
```

macOS: 12.6 Monterey
```
❯ rakudo --version
Welcome to Rakudo™ v2022.07.
Implementing the Raku® Programming Language v6.d.
Built on MoarVM version 2022.07.
```

**Comment by jnthn** [comment]
Fixed by #11 and https://github.com/croservices/cro-tls/commit/6a42be31ad88391e59fd4bf34837b62f875ee4f0.

**Comment by masukomi** [comment]
thanks @jnthn
